lib/galaxy_test/selenium/test_library_to_collections.py

```diff
@@ -55,7 +55,6 @@ def build_collection_and_assert(self):
 
     def collection_export(self, is_new_history=False, collection_option=None):
         random_name = self._get_random_name()
-        self.use_legacy_history()
```

Comment:
Seems to be a real issue also - replicated by hand. When new history is created dev for collection creation from libraries - it doesn't become the default history.

Reply:
Resolved in #13721.